src/test/kotlin/no/nav/syfo/TestDatabase.kt

```diff
@@ -0,0 +1,47 @@
+package no.nav.syfo
+
+import io.zonky.test.db.postgres.embedded.EmbeddedPostgres
+import no.nav.syfo.infrastructure.database.DatabaseInterface
+import org.flywaydb.core.Flyway
+import java.sql.Connection
+
+class TestDatabase : DatabaseInterface {
+    private val pg: EmbeddedPostgres = try {
+        EmbeddedPostgres.start()
+    } catch (e: Exception) {
+        EmbeddedPostgres.builder().setLocaleConfig("locale", "en_US").start()
+    }
+
+    override val connection: Connection
+        get() = pg.postgresDatabase.connection.apply { autoCommit = false }
+
+    init {
+
+        Flyway.configure().run {
+            dataSource(pg.postgresDatabase).load().migrate()
+        }
+    }
+
+    fun stop() {
+        pg.close()
+    }
+}
+
+fun DatabaseInterface.dropData() {
+    val queryList = emptyList<String>() // TODO: Delete statements for db-tables
```

Comment:
Er denne todoen viktig? 😊 Jeg skjønner ikke hva den betyr.
```suggestion
    val queryList = emptyList<String>()
```

Reply:
For å huske å legge til `DELETE FROM vurdering` osv her når vi har opprettet databasetabellene våre.